sherpa/plot/backends.py

```diff
@@ -612,7 +612,7 @@ def get_latex_for_string(self, txt):
         Returns
         -------
         latex : str
-            The text modified as appropriate for a backend so that the LaTeX
+            The test modified as appropriate for a backend so that the LaTeX
```

Comment:
Err, I think the original text (of `text`) was correct here. Or am I missing something?

Reply:
Force pushed to change back.